packages/components/src/stories/QueryModel.scss

```diff
@@ -0,0 +1,10 @@
+.form-row {
+    display: flex;
+}
+.form-row__input {
+    display: flex;
+    padding: 1em;
+}
+.form-row__input label {
+    padding: 0.5em;
+}
```

Comment:
Why are these added just for stories?

Reply:
They are used by the `ChangeableSchemaQuery` story in storybook and nowhere else.